### ISSUE TITLE
{"Fouttype":"OngeldigeSessieStatus","Omschrijving":null} while getting grades.

### ISSUE THREAD

**Issue by skillzzjesse** [simplyGits/MagisterJS#57]
This error weirdly just started popping up. Seen it discussed in the following two issues. Thought I should report it.
* https://github.com/lesander/magister-calendar/issues/12
* https://github.com/simplyGits/MagisterJS/issues/36

For the purpose of testing this I copied the code for getting grades from your examples page and altered it a bit to see where the error is caught.
```js
var Magister  = require('magister.js')

new Magister.Magister({
  school: {"url": "https://dspierson.magister.net"},
  username: "xxxx",
  password: "xxxx"
}).ready(function (err) {
  if (err) {
    console.log('login: '+err);
    return;
  }
  this.currentCourse(function (err, result) {
    if (err) {
      console.log('course: '+err);
      return;
    }
    result.grades(function (err, result) {
      if (err) {
        console.log('grades: '+err);
        return;
      }
      console.log(result.map(function (g) {
        return g.grade();
      }));
    });
  });
});
```

When running the following code in node this would be the result.
<img width="521" alt="screen shot 2016-11-13 at 20 52 28" src="https://cloud.githubusercontent.com/assets/15139826/20248391/88cb9bea-a9e3-11e6-8b66-abb280325de3.png">
**Note how** It's logged twice? 
*Testing this a couple of times actually resulted in random amounts of this error varying from 2 to 5*

**Comment by lieuwex** [comment]
I have no idea why this error comes up, is the official Magister site working fine for you? I see this error popping up a lot for some users today on my app too. Your API version is identical to mine, so it can't be an API change.
I will take a look if I can find out if this is an upstream bug, or something that I can fix.

> Note how It's logged twice?

Yup, I actually was talking about this with @tomsmeding this evening. :P
This has to do with the fill options, those are pretty buggy when set, and will call the callback multiple times when an error occurs. This is not a problem on m.js2 (again), but I will take a look if I can fix it for v1 currently.

**Comment by skillzzjesse** [comment]
Yeah, I tried logging into Magister manually right away. Didn't seem to have any problems but I didn't try out every part of the site (obviously). 

Let me know when there's an update. :)

**Comment by arunesh90** [comment]
I had this problem earlier tonight when using the website manually. It seems to be fixed now for me

**Comment by dannykruitbosch** [comment]
I've been getting the same error when trying to get the studyguide parts. Getting the studyguides is OK, but when I try to get the studyguide parts for one of the guides, it return the "OngeldigeSessieStatus" error.

**Comment by skillzzjesse** [comment]
Btw, I should also inform you that I am ofcourse running the latest version of magisterJS as of right now `v1.22.1`.

**Comment by skillzzjesse** [comment]
# update
I ran the test script I typed out above again this morning to check and the error did not show up anymore.
Looks like it's resolved? 😄

**Comment by skillzzjesse** [comment]
I noticed the weird error's are back and immediately went over to the 'official' Magister website for my school. Which returned an error the first time I tried: 
```
This site can't be reached, `dspierson.magister.net` took too long to respond.
```
However when I tried again it seemed to work fine.

The error's did not stop appearing afterwards though, they're back. I just thought I should report that.

**Comment by skillzzjesse** [comment]
Yup, I figured.
